java/java-guestbook/frontend/src/main/java/cloudcode/guestbook/frontend/FrontendController.java

```diff
@@ -31,9 +31,15 @@
     @GetMapping("/")
     public final String main(final Model model) {
         RestTemplate restTemplate = new RestTemplate();
-        GuestBookEntry[] response = restTemplate.getForObject(backendUri,
+        try {
+            GuestBookEntry[] response = restTemplate.getForObject(backendUri,
             GuestBookEntry[].class);
-        model.addAttribute("messages", response);
+            model.addAttribute("messages", response);
+        } catch(Exception e) {
+            System.out.println("Error retrieving messages from backend.");
```

Comment:
e.printStackTrace() please to get more details on connection

Reply:
Done